seed/migrations/0113_column_geocoding_order.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-10-09 14:16
+from __future__ import unicode_literals
+
+from django.db import migrations, models, transaction
+
+
+def forwards(apps, schema_editor):
+    Column = apps.get_model("seed", "Column")
+
+    with transaction.atomic():
+        # Default fields and order are those used before customization was enabled
+        default_geocoding_fields = [
+            'address_line_1',
+            'address_line_2',
+            'city',
+            'state',
+            'postal_code',
+        ]
+
+        # Start at 1 since 0 is considered deactivated.
+        for i, field in enumerate(default_geocoding_fields, start=1):
+            Column.objects.filter(
```

Comment:
These filters are an attempt to only target the original, DB address fields associated to each org's inventory types. I messed something like this up last time for default matching fields, so this deserves another pair of eyes.

Reply:
this looks right assuming you are adding 1,2,3,4,5 to address_line_1,address_line_2,city,state, postal_code for both taxlot and property fields. You have `is_extra_data` false which ensures that these are the database fields (and not extra data). Looks good!